docs/plugins/syslog.md

```diff
@@ -1,12 +1,12 @@
-## `syslog` plugin
+# `syslog` plugin
 
 The `syslog` plugin receives [syslog](https://en.wikipedia.org/wiki/Syslog) from network devices and outputs a parsed entry.
 
-### Configuration Fields
+## Configuration Fields
 
 | Field              | Default          | Description |                                                                                                                                                                                                  
 | ---                | ---              | ---         |                                                                                                                                                                                                     
-| `listen_port`      | `514`            | Network port to listen on                                                              |                                                     
+| `listen_port`      | `514`            | Network port to listen on. Default ports are 514 for UDP, 601 for TCP, and 6514 for TCP with TLS.|                                                     
```

Comment:
The default port description is misleading. The plugin does not have more than one default port. It might be nice to suggest this information to the user, but they need to set the port explicitly. If their device adheres to standards, and is sending syslog via TCP, then the port it uses by default is probably 6514 but this still needs to be set in the config for stanza.

Reply:
The modified examples bellow look great :+1:

Reply:
I gave a suggested rewording.

Reply:
Updated description per Dylan's suggestion.